starfish/intensity_table/intensity_table.py

```diff
@@ -419,49 +430,46 @@ def to_decoded_spots(self) -> DecodedSpots:
         df = df.drop(pixel_coordinates.intersection(df.columns), axis=1).drop(Features.AXIS, axis=1)
         return DecodedSpots(df)
 
-    def to_expression_matrix(self, regions: Optional[regional.many]=None) -> ExpressionMatrix:
+    def to_expression_matrix(self) -> ExpressionMatrix:
         """Generates a cell x gene count matrix where each cell is annotated with spatial metadata
 
-        Parameters
-        ----------
-        regions: Optional[regional.Many]
-            cell segmentation results that were used to assign points to cells. If not provided, the
-            centers of the cells will be estimated by taking the midpoint between the extreme-valued
-            spots on each axis.
+        Requires that spots in the IntensityTable have been assigned to cells.
 
         Returns
         -------
         ExpressionMatrix :
             cell x gene expression table
         """
-
-        # create the 2-d counts matrix
-        grouped = self.to_features_dataframe().groupby(['cell_id', 'target'])
+        try:
+            grouped = self.to_features_dataframe().groupby([Features.CELL_ID, Features.TARGET])
+        except KeyError as e:
```

Comment:
Rather than do this, why not just do:

```
if Features.CELL_ID not in self.coords:
    raise KeyError("IntensityTable must have 'cell_id' assignments for each cell before...")
```